### PR TITLE
add virt-top to nova roles

### PR DESCRIPTION
**Pull request by twaldrop**
Add virt-top to nova roles to assist in diagnostics and troubleshooting.

### CODE REVIEW

File: roles/nova-common/tasks/main.yml [hunk 1]
```diff
@@ -29,3 +29,6 @@
   tags:
     - logrotate
     - logging
+
+- name: install virt-top
+  apt: pkg=virt-top
```
Comment (j2sol) [inline roles/nova-common/tasks/main.yml]: Why not just add this to the list of packages we're installing up on line 6?

Comment (twaldrop) [inline roles/nova-common/tasks/main.yml]: Adding to the list of packages makes a lot of sense.  I'm not sure why I didn't do that in the first place.  PR update incoming

File: roles/nova-common/tasks/main.yml [hunk 2]
```diff
@@ -4,7 +4,7 @@
         createhome=yes generate_ssh_key=yes ssh_key_bits=4096
 
 - name: nova packages
-  apt: pkg=python-numpy
+  apt: pkg={{ nova.install_packages }}
```
Comment (j2sol) [inline roles/nova-common/tasks/main.yml]: that's not quite going to work. 
```
  apt:
    pkg: "{{ item }}"
  with_items: nova.install_packages
```

Something like that should do the trick. You still have to form it as a with_items type loop, but just feed it in the single variable which is a list of items.

File: roles/nova-common/tasks/main.yml [hunk 3]
```diff
@@ -4,7 +4,9 @@
         createhome=yes generate_ssh_key=yes ssh_key_bits=4096
 
 - name: nova packages
-  apt: pkg=python-numpy
+  apt: 
+    pkg="{{ item }}"
+  with_items: {{ nova.install_packages }}
```
Comment (j2sol) [inline roles/nova-common/tasks/main.yml]: You shouldn't have to use the mustache brackets here, just reference the variable.

**Pull request merged**